### PR TITLE
Update OAuth client list ui

### LINKED ISSUE #435: [Portal] Update OAuth client list UI
- [x] remove creation date column
- [x] change from icon button to action button
- [x] add remove button
- [x] handle update app config

### PR DESCRIPTION
**Pull request by axolotl0212**
base on #434

Closes #435 

- Update OAuth client list UI
- Added remove button

# Screenshot

Screenshot 2020-10-06 at 3 31 45 PM

Screenshot 2020-10-06 at 3 32 31 PM

### CODE REVIEW

File: portal/src/graphql/portal/OAuthClientConfigurationScreen.tsx [hunk 1]
```diff
@@ -63,10 +84,54 @@ function makeOAuthClientListColumns(
...
+    onDismiss,
+    removeOAuthClient,
+    clientId,
+    clientName,
+  } = props;
+  const { renderToString } = useContext(Context);
+
+  const onConfirm = useCallback(() => {
+    removeOAuthClient(clientId);
+    onDismiss();
```
Comment (kiootic) [inline portal/src/graphql/portal/OAuthClientConfigurationScreen.tsx:102]: We should keep the dialog open while the API request is in progress, similar to other dialogs. (e.g. disconnect identity)

**Comment by axolotl0212** [comment]
@kiootic updated, but the confirm button shifts upward while loading, investigating

**Review by kiootic — APPROVED** [review]
Testing seems fine, but found some problem with the loading button. I'll open another issue for that.

**Comment by axolotl0212** [comment]
> Testing seems fine, but found some problem with the loading button. I'll open another issue for that.

not the shifting upward problem?

**Comment by louischan-oursky** [comment]
@axolotl0212 Can you rebase?

**Comment by axolotl0212** [comment]
@louischan-oursky rebased

**Comment by louischan-oursky** [comment]
Nice!

**Pull request merged**